### PR TITLE
Fixes .values in QuerySet for translatable values

### PR DESCRIPTION
**Pull request by s-i-l-k-e**

### CODE REVIEW

**Comment by harrywhite4** [comment]
Nice one

File: README.md [hunk 1]
```diff
@@ -117,7 +117,7 @@ Pros:
 
 Cons:
 * You need to alter the models, so you can't make third-party libraries translatable.
-* It doesn't work on `queryset.values_list` - but we have a workaround below.
+* It doesn't work on `queryset.values_list` or `queryset.values` - but we have an easy workaround for both below.
```
Comment (LegoStormtroopr) [inline README.md:120]:
```suggestion
* It doesn't work on `queryset.values_list` or `queryset.values` natively - but we have an easy Queryset Mixin to add language support for both below.
```

File: garnett/mixins.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        cleaned_fields = []
+        for field in fields:
+            if isinstance(field, L):
+                expressions.update(
+                    {f"{PREFIX}{field.source_expressions[0].name}": field}
+                )
+            else:
+                cleaned_fields.append(field)
+
+        clone = super(TranslatedQuerySetMixin, self).values(
```
Comment (LegoStormtroopr) [inline garnett/mixins.py:59]:
```suggestion
        clone = super().values(
```

File: garnett/managers.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+from django.db.models import QuerySet
```
Comment (LegoStormtroopr) [inline garnett/managers.py:1]: BookQueryset should be in /tests/library

**Review by LegoStormtroopr — CHANGES_REQUESTED** [review]

File: garnett/managers.py (hunk 3 above)
Comment (s-i-l-k-e) [reply garnett/managers.py:1]: Can do

**Review by LegoStormtroopr — APPROVED** [review]

**Pull request merged**